src/components/List.js

```diff
@@ -0,0 +1,22 @@
+import Button from './Button';
+
+function List() {
+  const bookName = {
+    margin: '0',
+  };
+  const bookAuthor = {
+    marginTop: '0',
+  };
+
+  return (
+    <div>
+      <h2 style={bookName}>Book Name</h2>
+      <p style={bookAuthor}>Author</p>
+      <Button
+        type="button"
+        value="Delete"
+      />
+    </div>
```

Comment:
- [ ] `List.js` component is supposed to render a list of books using the `<Book>` component. 
Your component should render a list from an array of items using `map()` method.
Within the map method render a `<Book>` component which: 
- Takes the book data as a prop and renders the title and the author
- Has a button for removing books